README.md

```diff
@@ -28,7 +28,7 @@ Uploading a pipeline is done via the `fly set-pipeline` command, which is includ
 
 ### ci.cloud.gov
 
-Running pipelines on ci.cloud.gov is identical to running pipelines locally, with the exception that you will need to log in to ci.cloud.gov. Assuming you have permissions:
+Running pipelines on ci.cloud.gov is identical to running pipelines locally, with the exception that you will need to log in to ci.cloud.gov. Assuming you have permissions, here is how you would deploy the [ZAP pipeline](pipelines/zap/):
```

Comment:
Is it worth mentioning that this deploys from github's HEAD of the branch that you are on locally?   Could be confusing.